modules/test/playwright/pages/object-web/ViewObjectDefinitionsPage.ts

```diff
@@ -92,57 +94,77 @@ export class ViewObjectDefinitionsPage {
 		await this.deleteObjectDefinitionOption.click();
 	}
 
+	async clickAddObjectField(objectDefinitionLabel: string, fieldType: string, fieldLabel: string) {
+		this.clickEditObjectDefinitionLink(objectDefinitionLabel);
+
+		await this.page.getByRole('link', { name: 'Fields' }).click();
+
+		await this.page.getByRole('button', { name: 'Add Object Field' }).click();
+
+		await this.page.locator('input[name="label"]').fill(fieldLabel);
+
+		await this.page.getByText('Select an Option').click();
+
+		await this.page.getByRole('option', { name: `${fieldType}`, exact: true }).click();
+	}
+	
```

Comment:
maybe this should be in another page...investigate how the structure is presented today